src/TxHistory/TxHistoryNavigator.tsx

```diff
@@ -91,18 +91,10 @@ export const TxHistoryNavigator = () => {
         </Stack.Screen>
 
         <Stack.Screen //
-          name="send-select-token-from-list"
-          options={{
-            title: strings.selectAssetTitle,
-            ...sendOptions,
-          }}
-        >
-          {() => (
-            <Boundary>
-              <SelectTokenFromListScreen />
-            </Boundary>
-          )}
-        </Stack.Screen>
+          name="send"
+          component={SelectTokenFromListScreenNavigator}
```

Comment:
do we need a specific navigator for the bar? can't we use the current one?

Reply:
To be able to use the search provider https://github.com/Emurgo/yoroi-mobile/pull/2425/files#diff-21943f6c759fbcd7faab464e699efef813906ae31e3400e66a4a54c2426edceaR15

And the hooks.

Reply:
why not wrap the entire navigator? so the search could be used anywhere, and resetting the search when navigating into the context consumer?

Reply:
i will try

Reply:
it worked. aplied